service/src/main/java/bio/terra/landingzone/stairway/flight/create/resource/step/CreatePostgresqlDbStep.java

```diff
@@ -87,7 +87,7 @@ protected void createResource(FlightContext context, ArmManagers armManagers) {
                 .resourceId(postgres.id())
                 .resourceType(postgres.type())
                 .tags(postgres.tags())
-                .region(postgres.regionName())
+                .region(postgres.region().name())
```

Comment:
postgres.regionName() returns "South Central US", which results in a malformed URL.